config/locales/candidate_interface/restructured_work_history.yml

```diff
@@ -33,11 +33,11 @@ en:
       start_date:
         label: When did you start this job?
         hint_text: For example, 5 2018. If you cannot remember the exact month or year, enter an estimate.
```

Comment:
Do you think we should try and make it more clear in this hint text too? Any ideas @EmmaFrith 

Maybe something like `For example, 5 2018. If you cannot remember the exact month or year, you will still need to enter an estimate.` or something along those lines...

Reply:
Or maybe we could do the work for them and make one up, because whats the point of them entering a date which is completely made up if they really can't remember?

Reply:
@avinhurry interesting idea on making the hint text clearer..!

Agree we could rethink whether or not they even need to enter an estiamated month (and year) if they can't remember it, but currently in the work experience section we need both in order to be able to calculate whether there’s a "gap" of a month or more which we prompt users to explain...  However possibly we should rethink this policy, particularly for months and years that are a long time ago!

Reply:
Hmm, maybe we could avoid the calculation and just display that prompt and make it optional if the user doesn't enter a date or something along those lines, but yeah agree this needs more thought. Feels like enough users have been caught out here to change it a bit though